README.md

```diff
@@ -79,24 +79,10 @@ Autoloading is PSR-0 friendly.
 
 ### Via [Packagist](https://packagist.org/packages/ronanguilloux/isocodes) & [Composer](http://getcomposer.org/doc/00-intro.md)
 
-Create a composer.json file:
-
-```json
-{
-    "require": {"ronanguilloux/isocodes": "@stable"}
-}
-```
-
-Grab composer:
-
-```bash
-$ curl -s http://getcomposer.org/installer | php
-```
-
-Run install (will build the autoloading):
+Require the latest version of `ronanguilloux/isocodes` with Composer
 
 ```bash
-$ php composer.phar install
+composer require ronanguilloux/isocodes
 ```
 
```

Comment:
- Please leave the `Create a composer.json file` section as it is (lines 83 to 88): Someone may found the `@stable` flag helpful.
- OK to remove the `Grab composer ` section (lines 89 to 96)
- OK for the `Require the latest version` section